.github/workflows/release.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           output=$(curl --fail-with-body -sS \
             -H "Accept: application/vnd.github+json" \
             -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
-            "https://api.github.com/repos/$REPO/actions/workflows/postsubmit.yml/runs?per_page=1&branch=main&status=success")
+            "https://api.github.com/repos/$REPO/actions/workflows/postsubmit.yml/runs?per_page=1&branch=main&event=schedule&status=success")
```

Comment:
You could also filter for `event=workflow_dispatch` but I am not sure if the query parameter allows specifying multiple events combined by a logical `OR`.

Reply:
`event=schedule,workflow_dispatch` seems not to work so we'd have to do it with jq instead. The result of limiting to event=schedule is to require waiting a day to release new changes but we can live with this.